.github/workflows/bi-connectors.yml

```diff
@@ -11,40 +11,41 @@ jobs:
     - name: Prepare Power BI ODBC connector for 'Amazon OpenSearch Service'
       id: prep-powerbi-odbc-amz-os-svc
       run: |
+        mv OpenSearchProject-ODBC.m AmazonOpenSearchService-ODBC.m
         sed -i 's/<value>OpenSearch Project<\/value>/<value>Amazon OpenSearch Service<\/value>/g' resources.resx
-        sed -i 's/Documentation.Name = "OpenSearch Project"/Documentation.Name = "Amazon OpenSearch Service"/g' OpenSearch.m
-        sed -i 's/section OpenSearch;/section AmazonOpenSearchConnector;/' OpenSearch.m
-        sed -i 's/\[DataSource.Kind="OpenSearch", Publish="OpenSearch.Publish"\]/[DataSource.Kind="AmazonOpenSearchConnector", Publish="AmazonOpenSearchConnector.Publish"]/g' OpenSearch.m
-        sed -i 's/OpenSearch.Contents/AmazonOpenSearchConnector.Contents/g' OpenSearch.m
-        sed -i 's/OpenSearch.Publish/AmazonOpenSearchConnector.Publish/g' OpenSearch.m
-        sed -i 's/OpenSearch.Icons/AmazonOpenSearchConnector.Icons/g' OpenSearch.m
-        sed -i 's/OpenSearchImpl/AmazonOpenSearchConnectorImpl/g' OpenSearch.m
-        sed -i 's/OpenSearchType/AmazonOpenSearchConnectorType/g' OpenSearch.m
-        sed -i 's/OpenSearch = \[/AmazonOpenSearchConnector = [/' OpenSearch.m
-#        sed -i 's/"opensearch-sql"/"Amazon-OpenSearch-Service"/g' OpenSearch.m
+        sed -i 's/Documentation.Name = "OpenSearch Project"/Documentation.Name = "Amazon OpenSearch Service"/g' AmazonOpenSearchService-ODBC.m
+        sed -i 's/section #"OpenSearchProject-ODBC";/section #"AmazonOpenSearchService-ODBC";/' AmazonOpenSearchService-ODBC.m
+        sed -i 's/\[DataSource.Kind="OpenSearchProject-ODBC", Publish="OpenSearchProject-ODBC.Publish"\]/[DataSource.Kind="AmazonOpenSearchService-ODBC", Publish="AmazonOpenSearchService-ODBC.Publish"]/g' AmazonOpenSearchService-ODBC.m
+        sed -i 's/OpenSearchProject-ODBC.Contents/AmazonOpenSearchService-ODBC.Contents/g' AmazonOpenSearchService-ODBC.m
+        sed -i 's/OpenSearchProject-ODBC.Publish/AmazonOpenSearchService-ODBC.Publish/g' AmazonOpenSearchService-ODBC.m
+        sed -i 's/OpenSearchProject-ODBCImpl/AmazonOpenSearchService-ODBCImpl/g' AmazonOpenSearchService-ODBC.m
+        sed -i 's/OpenSearchProject-ODBCType/AmazonOpenSearchService-ODBCType/g' AmazonOpenSearchService-ODBC.m
+        sed -i 's/#"OpenSearchProject-ODBC" = \[/#"AmazonOpenSearchService-ODBC" = [/' AmazonOpenSearchService-ODBC.m
+        sed -i 's/"opensearchproject-odbc"/"amazonopensearchservice-odbc"/g' AmazonOpenSearchService-ODBC.m
+        sed -i 's/(dsr) => "OpenSearch Project SQL ODBC"/(dsr) => "Amazon OpenSearch Service SQL ODBC"/g' AmazonOpenSearchService-ODBC.m
       working-directory: bi-connectors/PowerBIConnector
     - name: Pack Power BI ODBC connector for 'Amazon OpenSearch Service'
       id: pack-powerbi-odbc-amz-os-svc
       run: |
-        zip AmazonOpenSearchService.mez *.png *.m *.resx *.pqm
+        zip AmazonOpenSearchService-ODBC.mez *.png *.m *.resx *.pqm
```

Comment:
Original filename was fine. It was the pq file content that needed to change.

Reply:
Addressed in https://github.com/Bit-Quill/opensearch-project-sql/pull/27/commits/f27402a988e667b46f62712520cd5337128a3008